extension/spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java

```diff
@@ -541,7 +494,7 @@ public static String randomBackend(SparkSettings sparkSettings , Logger logger)
      */
     @VisibleForTesting
     static List<PartitionDefinition> tabletsMapToPartition(Settings cfg, Map<String, List<Long>> be2Tablets,
-            String opaquedQueryPlan, String database, String table, Logger logger)
+                                                           String opaquedQueryPlan, String database, String table, Logger logger)
```

Comment:
The code style should be consistent

Reply:
Sorry😅, this `RestService.java` file shuld not been commited, it's my local file . Because of the version of my doris cluster is 0.12 which doesn't have `HTTPV2` API yet.
My fault!😇